trulens_eval/trulens_eval/streamlit.py

```diff
@@ -0,0 +1,274 @@
+import asyncio
+import json
+import math
+from typing import List
+
+# https://github.com/jerryjliu/llama_index/issues/7244:
+asyncio.set_event_loop(asyncio.new_event_loop())
+
+from millify import millify
+from pydantic import BaseModel
+import streamlit as st
+from streamlit_pills import pills
+
+from trulens_eval import Tru
+from trulens_eval.database.legacy.migration import MIGRATION_UNKNOWN_STR
+from trulens_eval.react_components.record_viewer import record_viewer
+from trulens_eval.schema.feedback import FeedbackCall
+from trulens_eval.schema.feedback import FeedbackDefinition
+from trulens_eval.schema.record import Record
+from trulens_eval.utils import display
+from trulens_eval.utils.python import Future
+from trulens_eval.ux import styles
+from trulens_eval.ux.components import draw_metadata
+from trulens_eval.ux.styles import CATEGORY
+
+tru = Tru()
+
+
+class FeedbackDisplay(BaseModel):
+    score: float = 0
+    calls: List[FeedbackCall]
+    icon: str
+
+
+def trulens_leaderboard(app_ids: List = None):
+    """
+    Render the leaderboard page.
+
+    Args:
+
+        app_ids : A list of application IDs (default is None)
+
+    !!! example
+
+        ```python
+        from trulens_eval import streamlit as trulens_st
+
+        trulens_st.trulens_leaderboard()
+        ```
+    """
+    tru = Tru()
+
+    lms = tru.db
```

Comment:
for my edification, what's `lms` stand for?

Reply:
not sure really. this is mostly copied code from `leaderboard.py`